packages/components/src/components/accordion/_accordion.scss

```diff
@@ -117,7 +118,7 @@
     margin: $accordion-arrow-margin;
     transform: rotate(90deg) #{'/*rtl:ignore*/'};
     transition: all $duration--fast-02 motion(standard, productive);
-    fill: $ui-05;
+    fill: $border-inverse;
```

Comment:
Is this for the chevron arrow? Should be using `$icon-primary`

Reply:
yeah, I think it was just not using the correct token originally. I was quite confused 😓

Reply:
Yeah I found a lot of places that were using the `$ui-05` color instead of the `$icon-01`. I think this is just because we made those icon tokens later and a lot of the code didn't get updated with the new token.